api/pkg/cli/hub/get_resource.go

```diff
@@ -365,8 +413,50 @@ func (rr *TektonHubResourceResult) MinPipelinesVersion() (string, error) {
 	return *rr.resourceData.LatestVersion.MinPipelinesVersion, nil
 }
 
-// MinPipelinesVersion returns the minimum pipeline version the resource is compatible
+// MinPipelinesVersion returns the minimum pipeline version the resource is compatible from Artifact Hub
 func (rr *ArtifactHubResourceResult) MinPipelinesVersion() (string, error) {
-	//TODO
-	return "", nil
+	if err := rr.validateData(); err != nil {
+		return "", err
+	}
+	resp := ArtifactHubPkgResponse{}
+	if err := json.Unmarshal(rr.data, &resp); err != nil {
+		return "", err
+	}
+	if resp.Data.PipelineMinVer == "" {
+		return "", fmt.Errorf("min pipeline version is not specified in the resource")
```

Comment:
```suggestion
		return "", fmt.Errorf("Min pipeline version is not specified in the resource")
```

Reply:
I think golang suggests that error string should not be capitalized, but it seems we used capitalized error strings across the codebase. I have updated to capitalized error strings  for consistency, in the future we could do a refactoring.